bin/docs/PCSX2.1

```diff
@@ -101,9 +101,9 @@ $XDG_HOME_DIR/pcsx2 or $HOME/.config/pcsx2
 All pcsx2 configurations and user datas.
 
 .SH "REPORTING BUGS"
-You can report bugs using the bug tracker at the pcsx2 google project, located at
+You can report bugs submitting an issue at the pcsx2 github repository, located at
 
-http://code.google.com/p/pcsx2
+https://github.com/PCSX2/pcsx2
```

Comment:
"github repository" can probably be changed to "Github issue tracker"

and the link to : https://github.com/PCSX2/pcsx2/issues